sdks/python/apache_beam/ml/transforms/handlers.py

```diff
@@ -84,14 +85,15 @@ def process(
       self,
       element,
   ):
+    hash_key, element = element
```

Comment:
how strict are we here for this module in terms of back compatibilities? If this module is designed for internal uses, could we add comments about no back-compatible guarantees?